crates/holochain/benches/consistency.rs

```diff
@@ -146,7 +146,10 @@ impl Consumer {
 async fn setup() -> (Producer, Consumer, Others) {
     let (tx, rx) = tokio::sync::mpsc::channel(1);
     let (dna, _, _) = SweetDnaFile::unique_from_test_wasms(vec![TestWasm::Anchor]).await;
-    let config = SweetConductorConfig::standard().no_publish();
+    let config = SweetConductorConfig::rendezvous(false)
+        .apply_shared_rendezvous()
```

Comment:
... that bootstrap bool is unfortunate. Without knowing the api, it looks like you're saying rendezvous false, and then applying it anyways.

Reply:
it's weird anyway, because "rendezvous" is not actually part of the config. So do two weirds make a non-weird?

Reply:
heh. well, we can iterate on this outside this pr